client/src/main/java/org/apache/rocketmq/client/impl/consumer/DefaultLitePullConsumerImpl.java

```diff
@@ -1036,6 +1026,25 @@ private void addResultToQueue(PullResult pullResult, ProcessQueue processQueue)
                     break;
             }
             updatePullOffset(messageQueue, pullResult.getNextBeginOffset(), processQueue);
+
+            if (!this.isCancelled()) {
+                scheduledThreadPoolExecutor.schedule(this, pullDelayTimeMills, TimeUnit.MILLISECONDS);
+            } else {
+                log.warn("The Pull Task is cancelled after doPullTask, {}", messageQueue);
+            }
+        }
+
+        private void handleError(Throwable e) {
+            if (e instanceof InterruptedException) {
+                log.warn("Polling thread was interrupted.", e);
+            } else {
+                if (e instanceof MQBrokerException && ((MQBrokerException) e).getResponseCode() == ResponseCode.FLOW_CONTROL) {
+                    pullDelayTimeMills = PULL_TIME_DELAY_MILLS_WHEN_BROKER_FLOW_CONTROL;
+                } else {
+                    pullDelayTimeMills = pullTimeDelayMillsWhenException;
```

Comment:
The time for the next scheduling is changed here, but there is no scheduling

Reply:
Thank you for pointing it out. I will correct it as soon as possible.

Reply:
> Thank you for pointing it out. I will correct it as soon as possible.

Go ahead! :-)